babel.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-typescript"]
+}
```

Comment:
I found out we were missing some preconstruct configuration

see:
- https://github.com/preconstruct/preconstruct/issues/483
- https://preconstruct.tools/guides/configuring-babel
- https://babeljs.io/docs/config-files#monorepos